code/Cargo.lock

```diff
@@ -653,6 +653,45 @@ dependencies = [
  "thiserror",
 ]
 
+[[package]]
+name = "beefy-light-client"
```

Comment:
why we depend on beefy client if we use grandpa?

Reply:
Because it's one of the clients in pallet-ibc